main/static/src/style/test.scss

```diff
@@ -0,0 +1,48 @@
+.test {
+  section {
+    padding: $line-height-computed 0;
+    > h2 {
+      margin-bottom: $line-height-computed;
+      border-bottom: 1px solid $hr-border-color;
+      a {
+        color: $headings-color;
+        text-decoration: none;
+      }
+
+      .on-hover {
+        display: none;
+      }
+      &:hover {
+        .on-hover {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4